lib/logger.js

```diff
@@ -226,15 +228,19 @@ Logger.prototype.log = function(statement, opts) {
         return this._err;
     }
 
-    this._bufferLog(message);
+    this._bufferLog(message, callback);
 };
 
-Logger.prototype._bufferLog = function(message) {
+Logger.prototype._bufferLog = function(message, callback) {
     if (!message || !message.line) {
         debug('Ignoring empty message');
         return;
     }
 
+    if (!callback || typeof callback !== 'function') {
+        callback = (err) => {debug(err);};
```

Comment:
Would prefer spaces between the brackets

Reply:
Another linting rule to add 
https://github.com/logdna/nodejs/blob/master/.eslintrc#L226
`"space-in-brackets": "always"`